src/main/java/edu/fiuba/algo3/modelo/Etapa/EtapaR.java

```diff
@@ -1,14 +1,18 @@
-package edu.fiuba.algo3.modelo;
+package edu.fiuba.algo3.modelo.Etapa;
 
 import edu.fiuba.algo3.modelo.juego.Jugador;
 import edu.fiuba.algo3.modelo.juego.Pais;
 
 public interface EtapaR {
```

Comment:
Noté que hay etapas que implementan ésta interfaz pero tiene varios métodos que se deja vacio, lo que pueden hacer es separar la interfaz en las que creen necesaria e que cada etapa implemente lo que necesite, si usa todos los métodos va a implementar todas las interfaces. El error es que se está violando el principio de Segregación de Interfaces